examples/npm_lib/src/js/interfaces/browser_js_spec_macros.rs

```diff
@@ -0,0 +1,14 @@
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(js_namespace = console)]
+    pub fn log(s: &str);
+}
+
+#[macro_export]
+macro_rules! console_log {
+    ($($t:tt)*) => {
+        ($crate::js::interfaces::browser_js_spec_macros::log(&format_args!($($t)*).to_string()))
+    }
+}
```

Comment:
Just use `gloo_console`? 

https://docs.rs/gloo-console/0.2.1/gloo_console/macro.log.html

Reply:
thanks yes i was able to replicate the same as
```rust
crate::console_log!("connect::kobold_npm_lib() {:?}", hash);
```
using 
```rust
log!(&format_args!("connect::kobold_npm_lib() {:?}", hash.as_ref().unwrap()).to_string());
```